lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -164,9 +164,108 @@ TF_DEFINE_PRIVATE_TOKENS(
     (vaddressmode)
     (filtertype)
     (channels)
+
+    // Texcoord reader identifiers:
+    (index)
+    (UV0)
+    (geompropvalue)
+    (ST_reader)
+    (vector2)
+);
+
+TF_DEFINE_PRIVATE_TOKENS(
+    _mtlxTopoNodeTokens,
+
+    // Topo affecting nodes due to object/model/world space parameter
+    (ND_position_vector3)
+    (ND_normal_vector3)
+    (ND_tangent_vector3)
+    (ND_bitangent_vector3)
+    // Topo affecting nodes due to channel index. We remap to geomprop in _AddMissingTexcoordReaders
+    (ND_texcoord_vector2)
+    (ND_texcoord_vector3)
+    // Color at vertices also affect topo, but we have not locked a naming scheme to go from index
+    // based to name based as we did for UV sets. We will mark them as topo-affecting, but there is
+    // nothing we can do to link them correctly to a primvar without specifying a naming scheme.
```

Comment:
Should we also specify a primvar naming scheme for these as well?

Reply:
Is there any scheme existing out there already?